tests/test_filequeue.py

```diff
@@ -104,16 +94,19 @@ async def testDoubleScan(self):
         # be one, since we didn't wait.
         os.link(tmp_file2, os.path.join(self.tmp_dir, os.path.basename(tmp_file2)))
         file_list = await fileq.dequeue_files()
-        self.assertEqual(len(file_list), 1)
-
-        await asyncio.sleep(3)
-        # we waited, and there should be two files
-        file_list = await fileq.dequeue_files()
         self.assertEqual(len(file_list), 2)
-
-        ret_file = file_list[0]
+        ret_file1 = file_list[0]
         ret_file2 = file_list[1]
-
-        os.unlink(ret_file)
+        os.unlink(ret_file1)
         os.unlink(ret_file2)
+
+        value = 451
+        try:
+            files = await asyncio.wait_for(fileq.dequeue_files(), timeout=3.0)
+            self.assertEqual(len(files), 0)
+        except Exception:
+            value = 123
+
+        self.assertEqual(value, 123)
```

Comment:
If I got it right, you're testing here if an exception was raised. I'm not familiar with `asynctest` library, but doesn't it provide a native way to do it, something similar to `assertRaises()` in `unittest`?

Reply:
I completely forgot about `assertRaises()` - I used that instead. 

Fixed.